Source/BugsnagConfiguration.h

```diff
@@ -69,7 +69,7 @@ typedef NSDictionary *_Nullable (^BugsnagBeforeNotifyHook)(
 /**
  *  The API key of a Bugsnag project
  */
-@property(readwrite, retain, nullable) NSString *apiKey;
+@property(readwrite, retain, nonnull) NSString *apiKey;
```

Comment:
is this a breaking change? Do we need to increment the major version number?

Reply:
This is a breaking change in Swift at least, as anyone fetching the value would need to update their code with the appropriate number of `?` and `!`. Its also not enforced, as `BugsnagConfiguration().apiKey` could be nil. 

We should not make this change just yet and instead design/think more about the interface, such as automatic API key detection from `Info.plist` or having a recommended constructor which takes an non-null API key as an argument. This change is also not required for this fix.